virtool_core/utils.py

```diff
@@ -68,7 +83,9 @@ def compress_file_with_pigz(path: Path, target: Path, processes: int):
     :param target: path where the compressed file should be stored
     :param processes: number of processes allowable for pigz (-p argument)
     """
-    command = ["pigz", "-p", str(processes), "-k", "--stdout", path]
+    processes_str: str = str(processes)
+    abs_path: str = os.path.abspath(path)
+    command = ["pigz", "-p", processes_str, "-k", "--stdout", abs_path]
```

Comment:
Use `pathlib` features.

https://stackoverflow.com/a/44569249
```suggestion
    command = ["pigz", "-p", processes_str, "-k", "--stdout", path.resolve()]
```